runtime/bindings/python/src/openvino/opset1/__init__.py

```diff
@@ -0,0 +1,111 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from ngraph.opset1.ops import absolute
+from ngraph.opset1.ops import absolute as abs
+from ngraph.opset1.ops import acos
+from ngraph.opset1.ops import add
+from ngraph.opset1.ops import asin
+from ngraph.opset1.ops import atan
+from ngraph.opset1.ops import avg_pool
+from ngraph.opset1.ops import batch_norm_inference
+from ngraph.opset1.ops import binary_convolution
+from ngraph.opset1.ops import broadcast
+from ngraph.opset1.ops import ceiling
+from ngraph.opset1.ops import ceiling as ceil
+from ngraph.opset1.ops import clamp
+from ngraph.opset1.ops import concat
+from ngraph.opset1.ops import constant
+from ngraph.opset1.ops import convert
+from ngraph.opset1.ops import convert_like
+from ngraph.opset1.ops import convolution
+from ngraph.opset1.ops import convolution_backprop_data
+from ngraph.opset1.ops import cos
+from ngraph.opset1.ops import cosh
+from ngraph.opset1.ops import ctc_greedy_decoder
+from ngraph.opset1.ops import deformable_convolution
+from ngraph.opset1.ops import deformable_psroi_pooling
+from ngraph.opset1.ops import depth_to_space
+from ngraph.opset1.ops import detection_output
```

Comment:
@ilyachur we have different naming style for IE and ngraph python API in new API

@akuporos should we align it?

Reply:
See https://www.python.org/dev/peps/pep-0008/

Reply:
I decided to add the new graph api in another PR.
Will be fixed there.